cmd/cli/kubectl-kyverno/oci/oci_pull.go

```diff
@@ -97,13 +96,9 @@ kyverno oci pull -i <imgref> -d policies`,
 						return fmt.Errorf("unmarshaling layer blob: %v", err)
 					}
 					for _, policy := range policies {
-						policyJsonBytes, err := json.Marshal(policy)
+						policyBytes, err := policyutils.ToYaml(policy)
```

Comment:
can we add a comment here , since function name says `ToYaml` but we doing `obj>json>yaml` ?

Reply:
@prateekpandey14 done.

Reply:
BTW, I added the comment on the `ToYaml` func, not here.